lib/blueprinter/extractors/auto_extractor.rb

```diff
@@ -19,7 +20,7 @@ def extract(field_name, object, local_options, options = {})
     private
 
     def default_value(field_options)
-      field_options.key?(:default) ? field_options.fetch(:default) : Blueprinter.configuration.field_default
+      field_options.key?(:default) ? field_options.fetch(:default) : @config_field_default
```

Comment:
Have you measured this particular optimization results? `Blueprinter.configuration.field_default` seems to be just a static call 🤔